programs/witness_node/main.cpp

```diff
@@ -155,7 +155,7 @@ int main(int argc, char** argv) {
    if (unhandled_exception)
    {
       elog("Exiting with error:\n${e}", ("e", unhandled_exception->to_detail_string()));
-      node->shutdown();
+      node->shutdown(false);
```

Comment:
I think this is not correct.

Please carefully evaluate all scenarios and the need of passing in different parameters. Best leave comments in code explaining why.

Reply:
sorry forgot here to check "replay-blockchain" mode for condition execution with different params